### PR TITLE
Using list-dir instead of os.walk to get a list of asset names in Que…

### PR DESCRIPTION
**Pull request by rahulj51**
…ryGenerator

Ref: Issue #1007

### CODE REVIEW

File: great_expectations/datasource/generator/query_generator.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def add_query(self, generator_asset, query):
 
     def get_available_data_asset_names(self):
         if self._queries_path:
-            defined_queries = [path for path in os.walk(self._queries_path) if str(path).endswith(".sql")]
+            defined_queries = [path for path in os.listdir(self._queries_path) if str(path).endswith(".sql")]
```
Comment (jcampbell) [inline great_expectations/datasource/generator/query_generator.py]:
```suggestion
            defined_queries = [path[:-4] for path in os.listdir(self._queries_path) if str(path).endswith(".sql")]
```

File: tests/datasource/generator/test_query_generator.py [hunk 2]
```diff
@@ -50,3 +56,17 @@ def test_partition_id():
...
+    # create queries path
+    context_path = empty_data_context.root_directory
+    safe_mmkdir(os.path.join(context_path, "datasources/mydatasource/generators/mygenerator/queries"))
+    shutil.copy("./tests/test_fixtures/dummy.sql", str(os.path.join(context_path, "datasources", "mydatasource",
+                                                                    "generators", "mygenerator", "queries")))
+
+    data_source = Datasource(name="mydatasource", data_context=empty_data_context)
+    generator = QueryGenerator(name="mygenerator", datasource=data_source)
+    sql_list = generator.get_available_data_asset_names()
+    assert "dummy.sql" in sql_list
```
Comment (jcampbell) [inline tests/datasource/generator/test_query_generator.py]:
```
    assert "dummyl" in sql_list
```

**Review by jcampbell — CHANGES_REQUESTED** [review]
This looks great! I made two small suggestions for consistency with other generator behavior.

**Comment by jcampbell** [comment]
@cla-bot check

File: great_expectations/datasource/generator/query_generator.py (hunk 1 above)
Comment (rahulj51) [reply great_expectations/datasource/generator/query_generator.py]: Thanks for the suggestion @jcampbell . Fixed.

**Review by jcampbell — APPROVED** [review]

**Pull request merged**